.github/scripts/parsers/1_21_5/items1215.mjs

```diff
@@ -53,7 +53,8 @@ export const Items1215 = {
                             value: item.nbt.SkullOwner.Properties.textures[0].Value
                         }
                     ]
-                } : undefined
+                } : undefined,
+                'minecraft:dyed_color': item.nbt.display.color,
```

Comment:
As to make just invalid data never gets included use ?. And || to undefined so null is never included